data/101/753/929/101753929.geojson

```diff
@@ -27,7 +27,7 @@
         "Chiajna"
     ],
     "name:eng_x_preferred":[
-        "Cheajna"
```

Comment:
"Cheajna" name would be stored as a "eng_x_variant" name in this case.

Reply:
@stepps00 should the `wof:name` also be updated when the eng_x_preferred name is updated?

Reply:
I would get rid of it altogether since "Cheajna" does not exist technically

Reply:
Should I remove the eng_x_variant altogether?

Reply:
I agree to disagree, I will add "Cheajna" as a variant, since it fixes my problem, but honestly that name doesn't exist with "e".